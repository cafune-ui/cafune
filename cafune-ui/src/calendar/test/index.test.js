import  Calendar  from '../';
import { render } from 'enzyme';
import { shallow, deep } from 'preact-render-spy';

describe('Calendar', () => {
  it('should render properly', () => {
    const wrapper = shallow(<Calendar />);
    expect(wrapper).toMatchSnapshot();

    wrapper.render(<Calendar defaultSelect={{ start: "2019/10/1", end: "2019/10/28"}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
