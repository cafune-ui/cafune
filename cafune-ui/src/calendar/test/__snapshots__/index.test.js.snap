// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar should render properly 1`] = `
RenderContext {
  "0": VNode {
    "attributes": undefined,
    "children": Array [],
    "key": undefined,
    "nodeName": [Function],
  },
  "length": 1,
}
`;

exports[`Calendar should render properly 2`] = `
RenderContext {
  "0": VNode {
    "attributes": Object {
      "defaultSelect": Object {
        "end": "2019/10/28",
        "start": "2019/10/1",
      },
    },
    "children": Array [],
    "key": undefined,
    "nodeName": [Function],
  },
  "length": 1,
}
`;
